lib/6to5/transformation/transform.js

```diff
@@ -68,6 +68,7 @@ _.each({
   computedPropertyNames:     require("./transformers/es6-computed-property-names"),
   destructuring:             require("./transformers/es6-destructuring"),
   defaultParameters:         require("./transformers/es6-default-parameters"),
+  forOfFast:                 require("./transformers/optional-for-of-fast"),
```

Comment:
The problem with that is that it won't override `forOf` when enabled, you'd have to blacklist `forOf` as well as enable `forOfFast`. Do you consider that preferable?